modules/manage/pages/audit-logging.adoc

```diff
@@ -0,0 +1,66 @@
+
+* `audit_enabled` : Boolean value to enable audit logging. When you set this to true, Redpanda checks for an existing topic named `__audit_log`. If none is found, Redpanda automatically creates one for you. (default: false)
+* `audit_log_num_partitions` : Integer value defining the number of partitions used by a newly created audit topic. This configuration applies only to the audit log topic and may be different from the cluster or other topic configurations. This cannot be altered for existing audit log topics. (default: 12)
+* `audit_log_replication_factor` : Integer value defining the replication factor for a newly created audit log topic. This configuration applies only to the audit log topic and may be different from the cluster or other topic configurations. This cannot be altered for existing audit log topics. (default: 3)
+* `audit_client_max_buffer_size` : Integer value defining the number of bytes allocated by the internal audit client for audit messages. When changing this, you must disable audit logging and then re-enable it for the change to take effect. Consider increasing this if your system generates a very large number of audit records in a short amount of time. (default: 16MiB)
+* `audit_queue_max_buffer_size_per_shard` : Integer value defining the maximum amount of memory used by the audit buffer in each shard. Once this size is reached, requests to log additional audit messages will return a non-retryable error. You must restart the cluster when changing this value. (default: 1MiB)
+* `audit_enabled_event_types` : List of strings in JSON style identifying the event types to include in the audit log. This may include any of the following - `management, produce, consume, describe, heartbeat, authenticate`. (default: `'["management","authenticate"]'`)
+* `audit_exclude_topics` : List of strings in JSON style identifying the topics the audit logging system should ignore. (default: null)
+* `audit_exclude_principals` : List of strings in JSON style identifying the principals the audit logging system should ignore. Principals can be listed as `User:name` or `name`, both are accepted. (default: null)
+
+Even though audited event messages are stored to a specialized immutable topic, standard topic settings still apply. For example, you can apply the same tiered storage, retention time, and replication settings available to normal topics. These particular options are important for controlling the amount of disk space utilized by your audit topics.
+
+IMPORTANT: Some audit logging properties need configured before first enabling audit logging as they impact creation of the `__audit_log` topic itself. These properties include: `audit_log_num_partitions` and `audit_log_replication_factor`. The Kafka API allows you to add partitions or alter the replication factor after enabling audit logging, but Redpanda prevents you from altering these two configuration values directly.
```

Comment:
```suggestion
IMPORTANT: You must configure certain audit logging properties before enabling audit logging because these settings impact the creation of the `__audit_log` topic itself. These properties include: `audit_log_num_partitions` and `audit_log_replication_factor`. The Kafka API allows you to add partitions or alter the replication factor after enabling audit logging, but Redpanda prevents you from altering these two configuration values directly.
```